### PR TITLE
[ko] 1st Korean localization work for v1.26

### LINKED ISSUE #38413: Korean Localization for dev-1.26-ko.1 branch
## In English

**This is a Feature Request**

Korean Localization for `dev-1.24-ko.2` branch.

**What would you like to be added**

Any localization contributions for Korean. 

**Comments**

- This issue is to notify the schedule of Korean localization to contributors.
- `dev-1.26-ko.1` is the current work branch for Korean localization.

**Important dates**
- A Korean localization work branch is merged to `main` branch after we complete around 3~4 weeks job.
- PR Freeze: **01/16** (PR opened after freeze will be requested to rebase to next work branch)
- Branch Merge: **01/19** (merge to main branch)

## In Korean

**기능 추가 요청 이슈**

`dev-1.26-ko.1` 브랜치에 대한 한글화 작업

**추가 필요 사항**

한글화에 관련된 모든 기여

**코멘트**

- 컨트리뷰터에게 쿠버네티스 문서 한글화에 대한 작업 스케줄 및 마일스톤을 공유하기 위한 이슈
- 현재 한글화 작업 브랜치는 [`dev-1.26-ko.1`](https://github.com/kubernetes/website/tree/dev-1.26-ko.1)

**주요 일정**

- 한글화 작업 브랜치는 약 2~3 주간 작업 후, main 브랜치에 머지 됨.
- PR 프리즈: **01/16** (작업 브랜치에 포함되는 PR을 확정, 이후의 PR은 차기 작업 브랜치로 리베이스 필요)
- 브랜치 머지: **01/19** (main 브랜치에 머지되며, k8s.io에 자동 반영)

/language ko
/help

### PR DESCRIPTION
**Pull request by seokho-son**
This PR is 1st Korean localization work for v1.26

 - [x] #38458
 - [x] #41518
 - [x] New Korean documents

Thanks to the contributors !

closes #38413

### CODE REVIEW

**Comment by seokho-son** [comment]
/hold

WIP. We need to complete #38458

**Comment by seokho-son** [comment]
Hi @jihoon-seo 
Could you open a PR to resolve the merge conflicts ? (open a PR targeting to dev-1.26-ko.1)
It is related with https://github.com/kubernetes/website/pull/39787.

I guess data/i18n/en/en.toml layouts/shortcodes/cve-feed.html have been changed after #39787

File: data/i18n/en/en.toml [hunk 1]
```diff
@@ -49,26 +49,17 @@ other = "CVE ID"
...
-
 [cve_summary]
 other = "Issue Summary"
 
 [cve_table]
 other = "Official Kubernetes CVE List"
 
-[cve_url]
-other = "CVE URL"
+[cve_table_date_format]
```
Comment (seokho-son) [inline data/i18n/en/en.toml:58]: Identical with the main branch HEAD.

Ex:
https://github.com/kubernetes/website/blob/main/data/i18n/en/en.toml#L58

File: layouts/shortcodes/cve-feed.html [hunk 2]
```diff
@@ -1,19 +1,23 @@
+{{ $feed := getJSON .Site.Params.cveFeedBucket }}
```
Comment (seokho-son) [inline layouts/shortcodes/cve-feed.html:1]: Identical with https://github.com/kubernetes/website/blob/main/layouts/shortcodes/cve-feed.html

File: layouts/_default/cve-feed.json [hunk 3]
```diff
@@ -1,23 +1 @@
...
-    {{ if $i }}, {{ end }}
-      {
-        {{ T "cve_json_id" | jsonify }}: {{ .cve_id | jsonify }},
-        {{ T "cve_json_url" | jsonify }}: {{ .issue_url | jsonify }},
-        {{ T "cve_json_external_url" | jsonify }}: {{ .cve_url | jsonify}},
-        {{ T "cve_json_summary" | jsonify }}: {{ replace (.summary | jsonify ) "\\u003e" ">"  }}
-      }{{ end }}
-   ]
-}
+{{ getJSON .Site.Params.cveFeedBucket | jsonify }}
```
Comment (seokho-son) [inline layouts/_default/cve-feed.json:1]: Identical with https://github.com/kubernetes/website/blob/main/layouts/_default/cve-feed.json

**Comment by seokho-son** [comment]
Hi @natalisucks @a-mccarthy :)

This PR is to merge Korean localization work for v1.26 into the main.
This PR needs additional approval from English approvers since the PR includes changes to data/i18n/en/en.toml and layouts/shortcodes/cve-feed.html which Korean approvers cannot approve. The changes here are just to sync the Korean development branch with the main branch in order to resolve merge conflicts in [PR: [ko] 1st Korean localization work for v1.26](https://github.com/kubernetes/website/pull/41322#41321). The changes to data/i18n/en/en.toml and layouts/shortcodes/cve-feed.html will be identical with the original in the latest main branch, and effective to dev-1.26-ko.1 branch only.

The commit related with the merge conflict in this PR was from https://github.com/kubernetes/website/pull/41322#issuecomment-1567657012 directly.

**Comment by natalisucks** [comment]
Happy to do so @seokho-son! 🚀 
/approve

**Pull request merged**